packages/elements/src/components/ino-chip/ino-chip.scss

```diff
@@ -17,8 +17,12 @@ $chip-color-font: theme.$primary;
 $icon-size: 16px;
 
 ino-chip {
-
   height: 24px; // default height
+  pointer-events: none;
+  
+  &.ino-chip--clickable {
+    pointer-events: all;
+  }
```

Comment:
Since `clickable` defaults to `true` I would suggest introducing a class `ino-chip--not-clickable` which disabled `pointer-events` if the class is set. Makes up for less code and improves readability IMO